src/main/java/mods/railcraft/common/blocks/wayobjects/RoutingLogic.java

```diff
@@ -380,8 +419,8 @@ public boolean matches(IRoutingTile tile, EntityMinecart cart) {
         }
 
         @Override
-        public boolean matches(IRoutingTile tile, EntityMinecart cart) {
-            return powered == tile.isPowered();
+        public int evaluate(IRoutingTile tile, EntityMinecart cart) {
+            return valueForBool(powered == tile.isPowered());
```

Comment:
Again, maintaining existing behavior--this is for `Redstone=true` and `Redstone=false`. I could see adding another keyword `RedstoneLevel` or something that does that.